File: store/src/main/java/com/kjj/store/dto/StoreStateDto.java
package com.kjj.store.dto;

import com.kjj.store.entity.StoreState;
import com.kjj.store.util.DateUtil;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class StoreStateDto {
    private String date;
    private String name;
    private Boolean off;

    public static StoreStateDto from(StoreState state) {
        return new StoreStateDto(
                DateUtil.makeLocaldateToFormatterString(state.getDate()),
                state.getName(),
                state.getOff()
        );
    }
}

File: store/src/main/kotlin/com/kjj/store/dto/StoreStateDto.kt
package com.kjj.store.dto

import com.kjj.store.entity.StoreState
import com.kjj.store.util.DateTool

data class StoreStateDto(
    val data: String,
    val name: String,
    val off: Boolean
) {
    companion object {
        fun from(state: StoreState): StoreStateDto {
            return StoreStateDto(
                DateTool.makeLocalDateToFormatterString(state.date),
                state.name,
                state.off
            )
        }
    }
}